flux-capacitor/src/model/algorithm.rs

```diff
@@ -0,0 +1,134 @@
+use async_graphql::dynamic::{
+    Field, FieldFuture, FieldValue, InputValue, Object, ResolverContext, TypeRef,
+};
+use async_graphql::{Context, FieldResult};
+use dynamic_graphql::internal::{OutputTypeName, Register, Registry, ResolveOwned, TypeName};
+use dynamic_graphql::SimpleObject;
+use raphtory::algorithms::pagerank::unweighted_page_rank;
+use raphtory::db::view_api::GraphViewOps;
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+pub(crate) struct Algorithms<G: GraphViewOps> {
+    graph: G,
+}
+
+impl<G: GraphViewOps> Algorithms<G> {
+    pub(crate) fn new(graph: &G) -> Self {
+        Self {
+            graph: graph.clone(),
+        }
+    }
+}
+
+impl<G: GraphViewOps> Register for Algorithms<G> {
+    fn register(registry: Registry) -> Registry {
+        let mut registry = registry;
+        let mut object = Object::new("Algorithms");
+
+        object = object.field(Field::new(
+            "numVerticesAlgo",
```

Comment:
Can we move the example to a separate package.

Reply:
removed it as we already have pagerank, was just for testing